components/sidebar/sidebar_channel/sidebar_channel_menu/sidebar_channel_menu.tsx

```diff
@@ -31,133 +31,191 @@ type Props = PropsFromRedux & OwnProps;
 
 const SidebarChannelMenu = (props: Props) => {
     const isLeaving = useRef(false);
```

Comment:
not part of this, but with this pattern you can safe some chars

```suggestion
    const {current: isLeaving} = useRef(false);
    ...
    // just use like this
    isLeaving = true;
```

works best with primitive values! 🚀

Reply:
Hey, Thank you for the feedback, but i would like to leave this as is. Hope you don't mind.